content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -31,29 +33,54 @@ you have selected.
 {{< feature-state for_k8s_version="v1.12" state="beta" >}}
```

Comment:
nit: Could we move this to the top after `### KMS v1` for consistency with the v2 stuff?

```suggestion
### KMS v1
{{< feature-state for_k8s_version="v1.12" state="beta" >}}

* Kubernetes version 1.10.0 or later is required

* Your cluster must use etcd v3 or later
```